### PR TITLE
Added host and port as cli flag

### PR DESCRIPTION
**Pull request by TakshPanchal**
/claim #141 

Added cli flags to environment variables directly using `argparse` package.

### CODE REVIEW

File: llmstack/cli.py [hunk 1]
```diff
@@ -3,16 +3,18 @@
...
 import sys
+import argparse
 from collections import defaultdict
 
 import docker
 import toml
 
 
-def run_django_command(command: list[str] = ['manage.py', 'runserver']):
+def run_django_command(command: list[str] = ["manage.py", "runserver"]):
```
Comment (vignesh-makerdojo) [inline llmstack/cli.py:13]: Please undo all the linter changes related to changing single quotes to double quotes.

File: llmstack/cli.py [hunk 2]
```diff
@@ -197,51 +231,69 @@ def signal_handler(sig, frame):
...
+            runner_environment[f"RUNNER_{key.upper()}"] = str(
+                config["llmstack-runner"][key]
+            )
+
+    # Load cli args
+    args = parser.parse_args()
+    if args.port is not None:
+        os.environ["LLMSTACK_PORT"] = args.port
+    if args.host is not None:
+        os.environ["HOST"] = args.host
```
Comment (vignesh-makerdojo) [inline llmstack/cli.py:248]: Where is this environ variable used? Can you please config if these changes work ?

**Review by vignesh-makerdojo — CHANGES_REQUESTED** [review]

**Comment by TakshPanchal** [comment]
@vignesh-makerdojo , I am having problems in setting-up the project, can you help me in how to set project up properly?

File: llmstack/cli.py (hunk 2 above)
Comment (TakshPanchal) [reply llmstack/cli.py:248]: All configs are loaded into env. variables, in line 306 `os.environ["LLMSTACK_PORT"]` is used as port.

**Comment by ajhai** [comment]
@TakshPanchal could you please rebase your branch, resolve merge conflicts and update the PR? Once the PR can be merged, I will merge this and close the bounty.

**Comment by TakshPanchal** [comment]
Okay give me some time, I will complete it by EOD.

**Comment by ajhai** [comment]
Thanks for working through this @TakshPanchal 

/approve

**Review by ajhai — APPROVED** [review]

**Review by ajhai — APPROVED** [review]

**Pull request merged**